ckanext/canada/recombinant_tables.yaml

```diff
@@ -697,6 +848,13 @@
     datastore_type: text
     xls_column_width: 20
 
+  - datastore_id: date
+    label: Date / Date
+    datastore_type: date
+    xls_column_width: 20
+    extract_date_year: true
+    extract_date_month: true
```

Comment:
can't we call this year? why are they year and month being extracted if this is just a year field?